app-modules/report/src/Filament/Widgets/StudentDeliverableTable.php

```diff
@@ -101,15 +102,61 @@ function (Builder $query): Builder {
             )
             ->columns([
                 TextColumn::make('full_name')
-                    ->label('Name'),
-                IconColumn::make('email_bounce')
-                    ->label('Email Eligibility')
-                    ->boolean()
-                    ->state(fn (Student $record): bool => ! $record->email_bounce),
-                IconColumn::make('sms_opt_out')
-                    ->label('SMS Eligibility')
-                    ->boolean()
-                    ->state(fn (Student $record): bool => ! $record->sms_opt_out),
-            ]);
+                    ->label('Name')
+                    ->searchable(),
+                TextColumn::make('primaryEmailAddress.address')
+                    ->label('Primary Email')
+                    ->searchable(),
+                TextColumn::make('email_bounce')
+                    ->label('Email Status')
+                    ->badge()
+                    ->color(fn (Student $record) => ($record->primaryEmailAddress?->bounced()->exists()) ? 'warning' : 'info')
+                    ->state(fn (Student $record) => ($record->primaryEmailAddress?->bounced()->exists()) ? 'Bounced' : 'Healthy'),
+                TextColumn::make('primaryPhoneNumber.number')
+                    ->label('Primary Phone')
+                    ->searchable(),
+                TextColumn::make('sms_opt_out')
+                    ->label('Phone Status')
+                    ->badge()
+                    ->color(fn (Student $record) => ($record->primaryPhoneNumber?->smsOptOut()->exists()) ? 'warning' : 'info')
+                    ->state(fn (Student $record) => ($record->primaryPhoneNumber?->smsOptOut()->exists()) ? 'Bounced' : 'Healthy'),
+            ])
+            ->filters([
+                SelectFilter::make('email_status')
+                    ->label('Email Status')
+                    ->options([
+                        'bounced' => 'Bounced',
+                        'healthy' => 'Healthy',
+                    ])
+                    ->query(function (Builder $query, array $data): Builder {
+                        if (($data['value']) === 'bounced') {
+                            return $query->whereHas('primaryEmailAddress.bounced');
+                        }
```

Comment:
I think it would be a good idea to do the same `($data['value'] ?? null)` here and in the health check that you do in the phone status filter.

Also, just a small nit, you could clean this all up by using `->when`s on the query instead of if statements.